internal/status/status.go

```diff
@@ -141,10 +151,11 @@ func (u *UpdateWriter) Send(update Update) {
 // isStatusEqual checks if two objects have equivalent status.
 //
 // Supported objects:
-//  GatewayClasses
-//  Gateway
-//  HTTPRoute
-//  TLSRoute
+//
+//	GatewayClasses
+//	Gateway
+//	HTTPRoute
+//	TLSRoute
```

Comment:
This is due to a change in gofmt from 1.18 to 1.19. This should be in a separate Go 1.19 PR.

Reply:
Reverted.